R/ERCCSpikeInConcentrations.R

```diff
@@ -14,21 +17,24 @@
 #' 
 #' @export
 #' @importFrom SingleCellExperiment splitAltExps
-ERCCSpikeInConcentrations <- function(volume_nl, dilution, genes, mix1 = TRUE) {
+ERCCSpikeInConcentrations <- function(genes = NULL, volume_nl = NULL, dilution = NULL) {
```

Comment:
No need for `volume_nl`, just have `volume` and use the documentation to mention what the unit is.